### ISSUE TITLE
Request form: labels communicate required data format

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2699]
### Issue description

As a user who relies on clear and accessible form instructions,
I want the form fields in the request form to have consistent and descriptive labels that communicate the expected data format,
so that I can input my information without confusion, ensuring the form fully complies with Section 508 accessibility standards.

### Acceptance Criteria

- [ ] Review the request form audit created as a part of #2866 
- [ ] Complete the A/C for #2872 
- [ ] Update the **.gov domain** page design to ensure that required fields (_What .gov domain do you want?_) are clearly marked as required.
- [ ] Complete A/C for #2877 
- [ ] Create a dev ticket to implement the changes.

### Additional Context

See[ slack thread](https://trussworks.slack.com/archives/C06UJSC6G91/p1724714079112069).

Here is an example of a an input field in the request form that fails `5.B Forms Descriptive Label`.
![Screenshot 2024-08-26 at 12 59 26 PM](https://github.com/user-attachments/assets/2f0b85e9-aa66-4b23-a012-5f4a9b22785a)

#### References for creating a dev ticket with accessibility requirements in the A/C
- #2871 
- #2849 
### Issue Links

Relates to Required fields in the request form are marked as required
[#2698](https://github.com/cisagov/manage.get.gov/issues/2698)

**Comment by gabydisarli** [comment]
@AnnaGingle I adjusted the language in this ticket to address fields in the domain management pages.

**Comment by AnnaGingle** [comment]
@gabydisarli I don't think I could get the domain management and request form done in one sprint. I will remove domain management for now.

**Comment by gabydisarli** [comment]
> @gabydisarli I don't think I could get the domain management and request form done in one sprint. I will remove domain management for now.

Sounds good to me!

**Comment by AnnaGingle** [comment]
Duplicate of #2698 . Closing
